src/go/k8s/kuttl-v2-test.yaml

```diff
@@ -19,7 +19,7 @@ commands:
       --set image.repository=localhost/redpanda-operator --namespace redpanda --create-namespace
       redpanda-operator redpanda/operator
 artifactsDir: tests/_e2e_artifacts_v2
-timeout: 360
+timeout: 720
```

Comment:
Why?

Reply:
I have hit issues when terminating and creating pods hitting timeouts before ever getting the second pod up. This has resolved the issue. I plan to add a few more tests soon so this also helps prevent issues with timeout in the near future. Unfortunately gracefully waiting for pandas to shutdown take a while, specially on upgrades.